params/config.go

```diff
@@ -147,10 +147,10 @@ var (
 		EIP150Hash:          common.HexToHash("0x9b095b36c15eaf13044373aef8ee0bd3a382a5abb92e402afa44b8249c3a90e9"),
 		EIP155Block:         big.NewInt(3),
 		EIP158Block:         big.NewInt(3),
```

Comment:
Why were some blocks set to `0`, but others left as non-zero values? Shouldn't everything be reset to `0`?

Reply:
From cursory glance changing it to zero is fine. We will be removing all the EIP references in the near future. https://github.com/dominant-strategies/go-quai/issues/431

Reply:
That wasn't the question. Just wondering why some were not changed to zero, but others were. Did you intentionally select some forks you want enabled?

Reply:
I misunderstood. All the numbers are changed to zero now.